templates/man.mustache

```diff
@@ -12,6 +12,12 @@
 
 	{{synopsis}}
 
+{{#has-alias}}
```

Comment:
This can be `{{#alias}}`

The `#` in Mustache is an operator for checking whether the variable is set.